Libraries/Components/ScrollResponder.js

```diff
@@ -112,6 +113,15 @@ type State = {
 };
 type Event = Object;
 
+function isTagInstanceOfTextInput(tag) {
+  var instance = getInstanceFromNode(tag);
+  return instance && (
+    instance.constructor.displayName === 'AndroidTextInput' ||
```

Comment:
Definitely not, `displayName` is a debugging information that may get stripped in production builds.